docs/content/guides/developer/getting-started/sui-install.mdx

```diff
@@ -112,7 +112,7 @@ To confirm that Sui installed correctly, type `sui --version` in your console or
 Run the following command to install Sui binaries from the `testnet` branch:
 
 ```shell
-cargo install --locked --git https://github.com/MystenLabs/sui.git --branch testnet sui
+cargo install --locked --git https://github.com/MystenLabs/sui.git --branch testnet sui --features tracing
```

Comment:
Let's add an explanation what tracing is and when / when not to use it. It does not hurt to be more doc complete here!